src/Symfony/Component/Console/Command/Command.php

```diff
@@ -516,7 +516,7 @@ public function getProcessedHelp()
     /**
      * Sets the aliases for the command.
      *
-     * @param string[] $aliases An array of aliases for the command
+     * @param string[]|\Traversable $aliases An array of aliases for the command
```

Comment:
I'm not familiar with this. I also prefer explicit types to. `string[]` not supporting `Traversable` could be ambiguous. 

Also difference when I get `array` and when I get `Traversable`.